source/gameengine/Converter/KX_BlenderSceneConverter.cpp

```diff
@@ -142,6 +142,11 @@ KX_BlenderSceneConverter::~KX_BlenderSceneConverter()
 		delete (adtList);
 	}
 
+	for (unsigned int i = 0; i < m_map_blenderobject_to_lodlevels.size(); i++) {
```

Comment:
can we create a typedef for unsigned types somewhere so it's shorter to type? Like this:
```c
typedef unsigned int uint;
typedef unsigned short ushort;
etc...
```